brainfuck/executor.go

```diff
@@ -0,0 +1,40 @@
+package brainfuck
+
+//Execute is the main function of program that takes as an argument
+//string in Brainfuck and execute every instruction from this string
+func Execute(programCode string) {
+
+	//currentStack contains slice for executing commands in loop
+	var currentStack = []startingLoop{}
```

Comment:
can be just `var currentStack []startingLoop`

Reply:
fixed